src/internal/kopia/inject/inject.go

```diff
@@ -15,6 +15,7 @@ type (
 	BackupConsumer interface {
 		ConsumeBackupCollections(
 			ctx context.Context,
+			backupReasons []kopia.Reason,
```

Comment:
Is the plan still to move the `Reasoner` interface out of kopia?  I think relocation will be needed to avoid circular dependencies, right?

Reply:
I think we can avoid circular dependencies for the moment even if `Reasoner` stays in the kopia package